bundles/com.espressif.idf.launch.serial.core/src/com/espressif/idf/launch/serial/internal/SerialFlashLaunchConfigDelegate.java

```diff
@@ -117,10 +117,12 @@ protected void launchInternal(ILaunchConfiguration configuration, String mode, I
 	{
 		IStringVariableManager varManager = VariablesPlugin.getDefault().getStringVariableManager();
 
-		String location = varManager.performStringSubstitution(IDFUtil.getIDFPythonEnvPath());
+		String location = configuration.getAttribute(ICDTLaunchConfigurationConstants.ATTR_LOCATION,
+				SYSTEM_PATH_PYTHON);
```

Comment:
Shouldn't the default value come from idf python env path or is it okay to give system path? 🤔

Reply:
Yes, will rely only on IDF_PYTHON_ENV for build and flash

Reply:
@kolipakakondal, @alirana01 removed system python path in the latest commit